### PR TITLE
increase the timeout time for esm tests to reduce test flakiness

### PR DESCRIPTION
**Pull request by khanayan123**
### What does this PR do?
increase the timeout time for esm tests to reduce test flakiness

### Motivation
esm tests failing on release branches

### CODE REVIEW

File: integration-tests/cucumber/cucumber.spec.js [hunk 1]
```diff
@@ -120,7 +126,7 @@ versions.forEach(version => {
           assert.equal(code, 0)
           done()
         })
-      })
+      }).tim
```
Comment (sabrenner) [inline integration-tests/cucumber/cucumber.spec.js:129]: should this be `timeout`?

**Review by sabrenner — CHANGES_REQUESTED** [review]

**Review by sabrenner — APPROVED** [review]

**Review by tlhunter — APPROVED** [review]

**Review by sabrenner — APPROVED** [review]

**Review by tlhunter — APPROVED** [review]

**Review by juan-fernandez — COMMENTED** [review]
are CI tests timing out because they're slow? Sometimes when actual failures happen they surface as timeouts. This change would just make it worse

**Review by juan-fernandez — APPROVED** [review]
I don't want to block if we think this is causing issues but I'd want to make sure the timeouts we see are actual timeouts or other issues that surface as timeouts. If it's the latter, this is just making it worse.

**Comment by khanayan123** [comment]
> I don't want to block if we think this is causing issues but I'd want to make sure the timeouts we see are actual timeouts or other issues that surface as timeouts. If it's the latter, this is just making it worse.

From my experience, I'd say the timeout errors are usually nothing more then just issues related to running tests in the CI where some runs are faster/slower then others and can cause timing issues. Usually anything deeper would be a bit more consistent rather then an occasional timeout failure. Although if you do feel strongly about it, we can not merge in the timeout change since the timeout errors usually only happen very infrequently and in those cases we can just rerun the job.

**Pull request merged**